src/pages/PeoplePage/HeaderLink.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { SortTypses } from '../../types/SortTypes';
+import { getSearchWith } from '../../utils/searchHelper';
+
+type Props = {
+  name: string;
+  sortType: SortTypses;
+};
+
+export const HeaderLink: React.FC<Props> = ({ name, sortType }) => {
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const getHeaderUrl = () => {
+    if (sort !== sortType) {
+      return getSearchWith(searchParams, { sort: sortType, order: null });
+    }
+
+    if (sort === sortType && !order) {
+      return getSearchWith(searchParams, { order: 'desc' });
```

Comment:
create enum for this
```suggestion
      return getSearchWith(searchParams, { order: OrderType.Desc });
```